tests/test_restore_names.py

```diff
@@ -1,3 +1,61 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_is_firstname_correct():
+    users = [
+        {
+            "first_name": None,
+            "last_name": "Holy",
+            "full_name": "Jack Holy",
+        },
+        {
+            "last_name": "Adams",
+            "full_name": "Mike Adams",
+        },
+    ]
+    restore_names(users)
+
+    assert users[0]["first_name"] == "Jack"
+    assert users[1]["first_name"] == "Mike"
```

Comment:
You can add `expected` variable that is correct answer, and you can only `assert users == expected`. If you do so you don't need last test.